src/Orleans.Runtime/GrainDirectory/LocalGrainDirectory.cs

```diff
@@ -135,8 +135,8 @@ internal class LocalGrainDirectory :
                     loggerFactory);
             GsiActivationMaintainer = new GlobalSingleInstanceActivationMaintainer(this, this.Logger, globalConfig, grainFactory, multiClusterOracle, executorService, loggerFactory);
 
-            var seedNode = grainMembershipOptions.Value.SeedNode?.ToSiloAddress();
-            this.seed = this.MyAddress.Endpoint.Equals(seedNode?.Endpoint) ? this.MyAddress : seedNode;
+            var primarySiloEndPoint = developmentOptions.Value.PrimarySiloEndPoint;
```

Comment:
developmentOptions name is ambiguous, since it sounds like it includes anything development related,  can we call it developmentMembershipOptions?

Reply:
I honestly preferred developmentOptions, but I'll change it if you think the longest is preferable